### PR TITLE
More matplotlib

### PR DESCRIPTION
**Pull request by jukent**
Attempt to get comments from @anissa111 #167 merged into foundations.

### CODE REVIEW

**Comment by jukent** [comment]
I wouldn't be surprised if the link to `additional-topics2` was broken, since that is in #230

**Comment by jukent** [comment]
Requesting review from all reviewers of #167

File: core/matplotlib/matplotlib-additional-topics-1.ipynb [hunk 1]
```diff
@@ -0,0 +1,2314 @@
+{
```
Comment (mgrover1) [inline core/matplotlib/matplotlib-additional-topics-1.ipynb:1]: <p>Can you show an example where you set normalize=False? Working with values that do not add up to 100?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/pythia-foundations/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Review by mgrover1 — CHANGES_REQUESTED** [review]
Looks good - just one minor comment I left on ReviewNB

File: core/matplotlib/matplotlib-additional-topics-1.ipynb (hunk 1 above)
Comment (jukent) [reply core/matplotlib/matplotlib-additional-topics-1.ipynb:1]: <p>Done!</p>

**Review by mgrover1 — APPROVED** [review]
Thanks for making those changes @jukent ! Good to go!!

**Pull request merged**